command/ibft/ibft.go

```diff
@@ -35,5 +36,7 @@ func registerSubcommands(baseCmd *cobra.Command) {
 		candidates.GetCommand(),
 		// ibft switch
 		_switch.GetCommand(),
+		//	ibft quorum
```

Comment:
Really a mega nitpick:
There is a space and a tab after the `//`. Please fix this formatting 🙏

Reply:
[Resolved](https://github.com/0xPolygon/polygon-edge/pull/549/commits/ce06e14ba253949d38f843f83ce1c43f7657d16f)